CMakeLists.txt

```diff
@@ -387,7 +387,7 @@ set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
 FetchContent_Populate(${extension_name}
   SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
   GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/slicersalt/${extension_name}
-  GIT_TAG        3d0d222f72f6340b3b672d4b6cd3b6aaa109902b # slicersalt-2019-10-02-4baa99c10
+  GIT_TAG        f147c9500b04fc978f28687be7796cb08f044b25 # slicersalt-2019-10-03-4baa99c10
```

Comment:
As a side note, there is no need to create a new branch if you only add a commit to the spharm-pdm branch.

`slicersalt-YYYY-MM-DD-SHA{9}` means that the base commit of the slicesalt branch is `SHA{9}` and it was made on `YYYY-MM-DD`.

See https://www.slicer.org/wiki/Documentation/Nightly/Developers/ProjectForks#Naming_convention_for_Slicer_fork_branches

Reply:
👍